python/cugraph/dask/opg_pagerank/pagerank.py

```diff
@@ -13,37 +13,120 @@
 # limitations under the License.
 #
 
-from cugraph.raft.dask.common.comms import Comms
-from cugraph.dask.common.input_utils import DistributedDataHandler
 from dask.distributed import wait, default_client
+from cugraph.dask.common.input_utils import get_local_data
 from cugraph.raft.dask.common.comms import worker_state
 from cugraph.opg.link_analysis import mg_pagerank_wrapper as mg_pagerank
+import warnings
 
 
-def common_func(sID, data, local_data):
-    print("Dataframe: ", data)
-    print("local data: ", local_data)
+def call_pagerank(sID, data, local_data, alpha, max_iter,
+                  tol, personalization, nstart):
     sessionstate = worker_state(sID)
-    mg_pagerank.mg_pagerank(data[0], local_data, sessionstate['handle'])
-    return 1
+    return mg_pagerank.mg_pagerank(data[0],
+                                   local_data,
+                                   sessionstate['wid'],
+                                   sessionstate['handle'],
+                                   alpha,
+                                   max_iter,
+                                   tol,
+                                   personalization,
+                                   nstart)
 
 
-def pagerank(input_graph):
-    print("INSIDE DASK PAGERANK")
+def pagerank(input_graph,
+             alpha=0.85,
+             personalization=None,
+             max_iter=100,
+             tol=1.0e-5,
+             nstart=None,
+             load_balance=True):
+
+    """
+    Find the PageRank values for each vertex in a graph using multiple GPUs.
+    cuGraph computes an approximation of the Pagerank using the power method.
+    The input graph must contain edge list as  dask-cudf dataframe with
+    one partition per GPU.
+
+    Parameters
+    ----------
+    graph : cugraph.Graph
```

Comment:
we support DiGraph too

Reply:
i think on the contrary, we should probably mention that we support only DiGraph. since we dont auto-symmetrize the distributed dataframe

Reply:
updated to mention that we currently accept only directed graphs for opg algorithms